internal/crypto/Cargo.toml

```diff
@@ -108,6 +108,11 @@ num-bigint-dig = "0.8.4"
 pkcs1 = "0.7.5"
 rsa = { version = "0.9.7", features = ["pem", "sha2"] }
 spki = "0.7.3"
+
+[target.'cfg(all(target_arch = "wasm32", not(target_os = "wasi")))'.dependencies]
```

Comment:
I think this whole dependency can go.

Reply:
I can do a wasm32-unknown-unknown cleanup later. Some of these dependencies are definitely still required.